bin/version-manager.js

```diff
@@ -55,6 +60,9 @@ function buildGlobs(cb) {
   const globs = []
   testGlobs.forEach((file) => {
     if (/(?:package\.json|\.tap\.js)$/.test(file)) {
+      if (/\.js$/.test(file)) {
+        patterns.push(path.basename(file))
+      }
```

Comment:
Does this handle the readme example case OK?

`versioned-tests --minor tests/versioned/*.tap.js`

Seems like it could end up grabbing `*.tap.js` as a pattern depending on how that `basename` behaves. The later pattern testing is doing an `indexOf` which would never match any files. Similarly, `*` is not wildcard in a regex match. May end up wanting to just glob in that situation. 

(Unless this does work just fine, hard to tell just but scanning code)

Reply:
Turning filenames into test patterns is only done *after* expanding glob patterns, so we're OK on this. Check it out:

```
$ npx versioned-tests tests/versioned/*.tap.js
// bin/version-manager.js:62 : console.log(file)
tests/versioned/amazon-dax-client.tap.js
tests/versioned/aws-sdk.tap.js
tests/versioned/dynamodb.tap.js
tests/versioned/http-services.tap.js
tests/versioned/instrumentation-supported.tap.js
tests/versioned/instrumentation-unsupported.tap.js
tests/versioned/s3.tap.js
tests/versioned/sns.tap.js
tests/versioned/sqs.tap.js
```

So all those filenames, which are expanded from `*.tap.js`, are used to match against test files. If a file matches at least one test pattern, we run it, so globbing works as expected: all files matching `*.tap.js` are run, but not `test-utils.js` and such.

Reply:
Ah, I see. I wasn't accounting for that happening on the way in. 

I wonder if you could avoid the pattern stuff unless there's only one file. Anyways, we don't need to keep thrashing on this. I just wish this module tended to be a bit more straightforward. Perhaps we can do some more cleanup at some point.

Reply:
There is definitely some architectural funk worthy of a refactor to fit the operating assumptions usage has born out, but I'll leave that for the future.